samples/SoundBoard/src/main/cpp/SoundBoardEngine.cpp

```diff
@@ -42,8 +42,12 @@ SoundBoardEngine::~SoundBoardEngine() {
     }
 }
 
-void SoundBoardEngine::tapSource(bool isDown, int32_t audioSource) {
-    mSynth->tapSource(isDown, audioSource);
+void SoundBoardEngine::setNoteOff(int32_t noteIndex) {
```

Comment:
noteOn() is not normally a "set" method because we are triggering an event, not setting/getting a property.

Reply:
Makes sense to me! Done!